packages/encryption/src/lib/encryption.ts

```diff
@@ -339,7 +342,7 @@ export const zipAndEncryptFiles = async (
     folder.file(files[i].name, files[i]);
   }
 
-  return encryptZip({ ...paramsBase, zip }, litNodeClient);
```

Comment:
Why remove `paramsBase` we still need to pass the AC conditions & `dataToEncrypt`?